AutomatticTracks/build.gradle

```diff
@@ -30,7 +30,6 @@ android {
     defaultConfig {
         minSdkVersion rootProject.minSdkVersion
         targetSdkVersion rootProject.targetSdkVersion
-        buildConfigField "String", "VERSION_NAME", "\"2.1.0\""
```

Comment:
Over few years this property wasn't used AFAIK. Also, if we'll spot a problem, we can always backtrack the Tracks version looking at the client.

Taking the correct version of the Tracks SDK is not straightforward - we assign version number during the build time based on git parameters. There's no API for that now, and it's not really planned. More context: pdnsEh-IK-p2#comment-1705

Reply:
Got it! Thanks for the context.
Perhaps if eventually we have a much wider use of Tracks (to observe things like version adoption, complaints, etc) it would make more sense to have the version.